workflow-service/src/main/java/com/redhat/parodos/workflow/execution/controller/WorkFlowController.java

```diff
@@ -124,6 +124,20 @@ public ResponseEntity<WorkFlowStatusResponseDTO> getStatus(@PathVariable String
 		return ResponseEntity.ok(workFlowService.getWorkFlowStatus(UUID.fromString(workFlowExecutionId)));
 	}
 
+	@Operation(summary = "Returns workflows' status by project id")
+	@ApiResponses(
+			value = {
+					@ApiResponse(responseCode = "200", description = "Succeeded",
+							content = { @Content(mediaType = "application/json",
+									schema = @Schema(implementation = List.class)) }),
+					@ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content),
+					@ApiResponse(responseCode = "403", description = "Forbidden", content = @Content) })
+	@GetMapping("/status")
+	public ResponseEntity<List<WorkFlowStatusResponseDTO>> getStatusByProjectId(
```

Comment:
not sure what is the reason for it, but the generated openapi.json returns a string as a result while I'd expect the type of the return value to be a list.

however, if the project is mandatory in this context, I wonder why not expose the project-status endpoint instead.

if the purpose is to evaluate the project status, then it seems more natural to query for the project resource instead of the workflow, as project status might contain other information relevant to the project's overall status.

What will be the use of the new endpoint?

Reply:
Since this method return the list of the statuses of all the workflows available in a project (if I understood correctly), maybe renaming the method into `getStatusesByProjectId` can ease the understanding.

Reply:
@gciavarrini @masayag new UI design has replaced the project Overview with workflows Overview.  Therefore project-status will not be needed any more, and instead we will need to show all workflow executions' info by project. I will update this pr to include info other than status in the return (e.g. startTeim,  endTime ...)